src/main/java/de/tum/in/www1/artemis/service/ConsistencyCheckService.java

```diff
@@ -81,6 +63,7 @@ public ConsistencyCheckService(Optional<VersionControlService> versionControlSer
      * @return List containing the resulting errors, if any.
      */
     public List<ConsistencyErrorDTO> checkConsistencyOfProgrammingExercise(ProgrammingExercise programmingExercise) {
+        log.debug("Checking consistency for programming exercise: {}", programmingExercise.getId());
```

Comment:
Don't think we need this log as we already log in the rest endpoint. What do you think?

Reply:
Makes sense yes
Thanks 🙂